TenancyInformationApi/V1/Factories/TenancyResponseFactory.cs

```diff
@@ -0,0 +1,29 @@
+using System.Globalization;
+using TenancyInformationApi.V1.Boundary.Response;
+using TenancyInformationApi.V1.Domain;
+
+namespace TenancyInformationApi.V1.Factories
+{
+    public static class TenancyResponseFactory
+    {
+        public static TenancyInformationResponse ToResponse(this Tenancy tenancy)
+        {
+            return new TenancyInformationResponse
+            {
```

Comment:
If any of the fields which are being converted to string are not required fields then they will need a ? before ToString to stop it throwing